### PR TITLE
Use cross-spawn in tests so that they won't fail on Windows.

### PR DESCRIPTION
**Pull request by fatfisz**
Quite self-explanatory :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   ],
   "dependencies": {
     "async": "^1.2.1",
+    "cross-spawn": "^2.0.0",
```
Comment (kevva) [inline package.json]: This should be a `devDependency`.

File: test/test.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 var assert = require('assert');
 var fs = require('fs');
 var path = require('path');
-var spawn = require('child_process').spawn;
+var spawn = require('cross-spawn');
```
Comment (kevva) [inline test/test.js]: Move this to the end of the list since it isn't a core module (core modules are first).

File: package.json (hunk 1 above)
Comment (fatfisz) [inline package.json]: Yep, mistyped that.

**Comment by fatfisz** [comment]
Should I squash the commits?

**Comment by kevva** [comment]
Amazing that this issue still exists tbh.

**Pull request closed without merging**